client/src/Components/ReviewsContainer/ReviewsContainer.jsx

```diff
@@ -1,13 +1,25 @@
 import React from "react"
+import Container from "react-bootstrap/Container"
+import Row from "react-bootstrap/Row"
+import Col from "react-bootstrap/Col"
 import useClickLogger from "../../hooks/useClickLogger.jsx"
-import ReviewList from "./List/ReviewList.jsx"
+import Breakdowns from "./Breakdowns/Breakdowns.jsx"
+import List from "./List/List.jsx"
 import "bootstrap/dist/css/bootstrap.min.css"
```

Comment:
we should be able to have one import in index.jsx.

```js
// src/index.jsx
import React from "react"
import "bootstrap/dist/css/bootstrap.min.css"

...
ReactDOM.render(<Component />)
```

Reply:
will bring this up in team meeting